metadata.json

```diff
@@ -64,6 +64,12 @@
       "operatingsystem": "SLED"
     },
     {
+      "operatingsystem": "OpenSuSE",
```

Comment:
Could you post a link to a gist of your entire facter output? This would clear it up for me as well as be useful in the future, since I don't have an OpenSuSE 13 system.